webrtc-audio-processing-sys/src/lib.rs

```diff
@@ -4,6 +4,9 @@
 // https://github.com/rust-lang/rust-bindgen/issues/1651
 #![allow(deref_nullptr)]
 
+#[allow(unused_imports)]
+use std::ptr::null;
```

Comment:
This seems to be only used in test -> I think we can move the statement to be under `mod tests {`.